### PR TITLE
Add Refresh Terminologies Cache Functionality

### LINKED ISSUE #118: Enable clearing of Terminology cache
Currently the local terminology cache cannot be manually cleared.

### PR DESCRIPTION
**Pull request by fschrader1992**
This PR:

- Adds a function calling odml terminologies refresh to update cache files. Closes #118

### CODE REVIEW

**Comment by fschrader1992** [comment]
Checks for Code Quality:
- [x] Pylint
- [x] Rebase onto GNODE Master
- [x] Test with Python3.8

**Review by mpsonntag — CHANGES_REQUESTED** [review]
The current implementation locks the editor until the refresh is done basically freezing the application for the user without feedback.

It would be better to run the actual refresh in the background.

File: odmlui/editor.py [hunk 1]
```diff
@@ -950,7 +952,8 @@ def quit(self, action, extra=None):
     def on_refresh_cache(self, action):
         url = self.current_tab.document.repository
         if url:
-            terminology.refresh(url)
+            thread = threading.Thread(target=lambda rep_url: terminology.refresh, args=(url,))
```
Comment (mpsonntag) [inline odmlui/editor.py]: This does not work as expected. Please test properly before pushing a commit.

**Review by mpsonntag — CHANGES_REQUESTED** [review]

**Comment by fschrader1992** [comment]
Checks for Code Quality:
- [x] Pylint
- [x] Rebase onto GNODE Master
- [x] Test with Python3.8

**Pull request merged**